README.md

```diff
@@ -6,8 +6,7 @@ Building the site requires [Jekyll](http://jekyllrb.com/docs) 3.3.0 or newer.
 The easiest way to install jekyll is via a Ruby Gem. This will create a directory called `site` 
 containing `index.html` as well as the rest of the compiled directories and files.
 
-To install Jekyll and its required dependencies, execute `sudo gem install jekyll pygments.rb` 
-and `sudo pip install Pygments`.
+To install Jekyll and its required dependencies, execute `sudo gem install jekyll rogue`.
```

Comment:
Oops, it's rouge rather than rogue. Others are fixing it in the Docker image. I'll fix this.

Reply:
Argh ..